### ISSUE TITLE
Add integration test case for "Order by color"

### ISSUE THREAD

**Issue by ericcornelissen** [simple-icons/simple-icons-website#82]
### Kind of issue 

- [ ] Feature
- [ ] Bug
- [ ] Design
- [x] Other, namely: Testing

### Description

Relates to #66.

We should add an integration test verifying that the "Order by color" button changes the ordering of the icons. There already exists [a prototype test case](https://github.com/simple-icons/simple-icons-website/blob/ea9452ce56c77f80dfaff4ecf99de22ccd192859/tests/integration.test.js#L297-L307) for this, but it's skipped as it currently doesn't work.

**Comment by jorgeamadosoria** [comment]
will address this as part of #110

**Comment by ericcornelissen** [comment]
Why not as a separate PR? Adding it to #110 will delay it getting merged.

**Comment by jorgeamadosoria** [comment]
Yes, but I feel they belong together, you know? One is code and the other is the testing for said code. I would feel better having something that is tested.

**Comment by ericcornelissen** [comment]
I see.

However, depending on what your test will look like this may actually not be relevant. If you're just verifying the order matches what is returned by the color sorting algorithm, then you're not actually testing if the bug is fixed just that color sorting does "something". This is why I would suggest separating them: the bugfix is out sooner and won't by delayed by a test that doesn't actually test if the bug is fixed.

Of course, if you have a plan for actually testing if the bug is fixed, it's definitely better to include it in the PR. And if you prefer to keep it together, by all means :slightly_smiling_face:

**Comment by jorgeamadosoria** [comment]
I wasn't planning on having a test to see if the bug is fixed, but now that you mention it, it does make sense. I will do that in addition to the standard "just check if they are sorted" test. Thanks for the suggestion.

**Comment by ericcornelissen** [comment]
[The test case](https://github.com/simple-icons/simple-icons-website/blob/7843220e0a2b825dffc2e3f58fb9d41fae7e0e5b/tests/e2e.test.js#L297) is still skipped after #110, what do you think about this issue @jorgeamadosoria?

**Comment by jorgeamadosoria** [comment]
> [The test case](https://github.com/simple-icons/simple-icons-website/blob/7843220e0a2b825dffc2e3f58fb9d41fae7e0e5b/tests/e2e.test.js#L297) is still skipped after #110, what do you think about this issue @jorgeamadosoria?

the test case is impossible to implement unless / until we get around to reorder HTML elements.
Since that's going to be necessary for color ordering for nonJS users anyways, maybe after implementing the static page for that ticket the test case would make more sense.
Right now, and as discussed on #110, there is nothing to be done to unskip it.
